API-RUNNER.py

```diff
@@ -0,0 +1,64 @@
+import json
+import time
+import schedule
+import requests
+import subprocess
+from dotenv import dotenv_values
+
+config = dotenv_values(".env")
+
+def runAuthServiceKarate():
+    subprocess.run('./bin/auth-service/karate', shell=True)
+
+def sendReport():
+    log = "bashupload.log"
+    with open(log) as f:
+        bashlog = f.readlines()
+
+    reportUrl = bashlog[0]
+    filename = "./tests/karate/karate-0.9.6/target/karate-reports/karate-summary-json.txt" #TO DO
```

Comment:
Why is there a "#TO DO" at the end ?

Reply:
Because in the future this service will handle few different services. So I will use configuration files to know the path of the summary and not using a hard coded path.

But I will edit the comment to be more understandable ! Thx